### ISSUE TITLE
Unable to update minNodeCount for nodePool as cluster parent

### ISSUE THREAD

**Issue by praveenperera** [pulumi/pulumi-google-native#404]
## Hello!

- Vote on this issue by adding a 👍 reaction
- To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already)

## Issue Details

I am defining the `nodePools` as field in `new googleNative.container.v1.Cluster`

in the node pool I'm changing

```diff
-    autoscaling: { enabled: true, minNodeCount: 1, maxNodeCount: 10 },
+    autoscaling: { enabled: true, minNodeCount: 0, maxNodeCount: 10 },
```

I get the error:

` error: error sending request: googleapi: Error 400: Must provide an update.: "https://container.googleapis.com/v1/projects/cloud-project-system-dev/locations/us-central1/clusters/my-cluster-dev" map[name:my-cluster-dev-dev projectId:cloud-project-system-dev]`

**Comment by praveenperera** [comment]
I already recreated the cluster but I'm using latest google-native package and pulumi 3.26.1.

Steps to reproduce is literally creating a cluster with one node pool `minNodeCount: 1` and then latest changing `minNodeCount: 0`

**Comment by viveklak** [comment]
This is fixed in #588
